src/python-common/ceph/deployment/drive_group.py

```diff
@@ -108,28 +111,6 @@ def __init__(self, msg):
         super(DriveGroupValidationError, self).__init__('Failed to validate Drive Group: ' + msg)
 
 
-class DriveGroupSpecs(object):
-    """ Container class to parse drivegroups """
-
-    def __init__(self, drive_group_json):
-        # type: (list) -> None
-        self.drive_group_json = drive_group_json
-
-        if isinstance(self.drive_group_json, dict):
-            # from legacy representation (till Octopus)
```

Comment:
if you remove this, you have to make sure that everyone uses the new representation.

Reply:
I'd say we didn't have anyone using this before octopus (I recall the actual drivegroups were just implemented in octopus)

Reply:
and I guess we should while we don't have any active users ;)